assembly/std/index.ts

```diff
@@ -219,35 +219,43 @@ export function getOpKeys(): Array<StaticArray<u8>> {
  *
```

Comment:
prefix param description is missing, can you explain how work the prefix filter?